assets/js/script.js

```diff
@@ -1,12 +1,66 @@
-document.getElementById('add-button').addEventListener('click', function() {
+document.getElementById('add-button').addEventListener('click', function () {
     const input = document.getElementById('item-input');
     const listContainer = document.getElementById('list-container');
     const listItem = document.createElement('li');
 
     if (input.value.trim() !== '') {
         listItem.innerHTML = `${input.value} <button class="delete is-small"> </button>`;
         listContainer.appendChild(listItem);
-        input.value = ''; 
+        input.value = '';
 
     }
 });
+
+// Modal JS Functionality Implementation Below:
+
+// Clicking the 'x' button in the top right: 
+document.addEventListener('DOMContentLoaded', function () {
+    const modal = document.querySelector('.modal');
+    const modalButton = document.querySelector('.button');
+    const closeButton = modal.querySelector('.delete');
+
+    // Fx to open the modal
+    function openModal() {
+        modal.classList.add('is-active');
+    }
+
+    // Fx to close the modal
+    function closeModal() {
+        modal.classList.remove('is-active');
+    }
+
+    // Event listener for the button to open the modal
+    modalButton.addEventListener('click', closeModal);
```

Comment:
I noticed your comment for this code is to "open the modal", but after 'click' it says closeModal. Is this working how you want it to?

Reply:
Nice catch! This portion of my syntax was a typo. The code I wrote beneath (on line 34) is the correct one. However, with this still typed in, the modal JS functionality was still working how I wanted it to. I made sure to run everything + test it to make sure it works prior to making the pull request. I'm not sure if it's because the second Event Listener that I added overrode it or what, but I deleted this line of code since it is unnecessary.